Source/WebCore/animation/CSSAnimation.cpp

```diff
@@ -168,7 +168,10 @@ void CSSAnimation::syncStyleOriginatedTimeline()
         }, [&] (const Animation::AnonymousViewTimeline& anonymousViewTimeline) {
             auto insets = anonymousViewTimeline.insets;
             auto viewTimeline = ViewTimeline::create(nullAtom(), anonymousViewTimeline.axis, WTFMove(insets));
-            viewTimeline->setSubject(target.ptr());
+            if (auto owningElement = this->owningElement())
+                viewTimeline->setSubject(*owningElement);
+            else
+                viewTimeline->setSubject(nullptr);
```

Comment:
We have an `ASSERT(owningElement())` at the start of this method, so presumably we shouldn't enter the `else` branch here. We should also fix the similar code for scroll timelines.